test/Algebra/Graph/Test/Generic.hs

```diff
@@ -1515,6 +1515,16 @@ testInduce (prefix, API{..}) = do
     test "isSubgraphOf (induce p x) x == True" $ \(apply -> p) x ->
           isSubgraphOf (induce p x) x == True
 
+testInduceJust :: Testsuite g Ord -> IO ()
```

Comment:
I think we should include this test into `testTransformations` above. Then it will be called from all testsuites.

Reply:
I wanted to do it initially but I realized that I can't implement `induceJust` for `AdjacencyIntMap`. Is there any workaround?

Reply:
Oh, good point. In this case let's stick to your current approach.